pse/p0tionConfig.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Ethstorage Circuits Ceremony",
+  "description": "This is a trusted setup ceremony for the Ethstorage Circuits with is used to proof storage on L2 dynamic datasets with an Ethereum L1 Contract",
+  "startDate": "2025-07-18T00:00:00",
+  "endDate": "2025-08-01T00:00:00",
```

Comment:
Is this intended for a production run or just a test?

Based on [this](https://github.com/privacy-scaling-explorations/DefinitelySetup/blob/main/ceremonies/README.md#process), I assume we can start with a test run to ensure everything works as expected.

If it’s just a test, I think the end time might be unnecessarily long.

Reply:
This PR will apply to [DefinitelySetup staging branch](https://github.com/privacy-scaling-explorations/DefinitelySetup/blob/staging/ceremonies/ethstorage-trusted-setup-ceremony) for test;
And I will update the timeline for main branch later.